content/cloud/data-postgres.md

```diff
@@ -16,7 +16,7 @@ url = "https://github.com/fermyon/developer/blob/main/content/cloud/data-postgre
 
 ## PostgreSQL
 
-[PostgreSQL](https://www.postgresql.org/), is a powerful, open-source object-relational database system that has earned a strong reputation for reliability, robustness and performance. This tutorial will implement a persistent storage solution for Fermyon Cloud, using PostgreSQL. In this tutorial, we will be using [ElephantSQL](https://www.elephantsql.com/plans.html)'s free PostgreSQL service.
+[PostgreSQL](https://www.postgresql.org/), is a powerful, open-source object-relational database system that has earned a strong reputation for reliability, robustness and performance. This tutorial will implement a persistent storage solution for Fermyon Cloud, using PostgreSQL. In this tutorial, we will be using ElephantSQL's free PostgreSQL service.
```

Comment:
ElephantSQL is discontinued, and we shouldn't refer to it.  (I _thought_ I had addressed this somewhere but I can't find a PR so... I guess I didn't.)  I realise this turns the edit into a rather bigger research and conversion exercise, though, so feel free to raise a separate issue and punt on it.